src/couch_replicator_httpc.erl

```diff
@@ -406,3 +422,33 @@ after_redirect(RedirectUrl, _Code, HttpDb, Params) ->
 after_redirect(RedirectUrl, HttpDb, Params) ->
     Params2 = lists:keydelete(path, 1, lists:keydelete(qs, 1, Params)),
     {HttpDb#httpdb{url = RedirectUrl}, Params2}.
+
+
+backoff_key(HttpDb, Params) ->
+    Method = get_value(method, Params, get),
+    Url = HttpDb#httpdb.url,
+    {Url, Method}.
```

Comment:
The rate limiter works based on requests, but in general could be anything that identifies a participant. In our case, I picked url + method. Those can be duplicates, as can have multiple replications with the same source for example.